### ISSUE TITLE
Wrong Color on my Monitor

### ISSUE THREAD

**Issue by MH0386** [elementary/triage#266]
### Discussed in https://github.com/orgs/elementary/discussions/210

<div type='discussions-op-text'>

<sup>Originally posted by **Mashrafi2006** April 5, 2023</sup>

I recently installed Elementary OS and noticed that the colors on my monitor don't look right. It seems like the operating system is using the wrong color profile. I'm not sure how to change the color profile to make it look correct.

Can someone help me with this issue? How can I change the color profile on Elementary OS to make it use the correct one for my monitor? 

Thanks in advance for your help.
</div>

**Comment by MH0386** [comment]
Hi everyone, 

I also have the same problem. My screen has a weird colour. I tried to install nvidia drivers, but I can't due to X server. the elementary os just log out When I kill Xorg. I tried to install all drivers from the AppCenter, but there is no change. 

Thanks in advance

![RDT_20230707_2006021504696373876245639.jpg](https://github.com/elementary/triage/assets/77013511/698a0166-ae71-4d88-a853-bfd21cf2f20c)

GPU: GTX 1660
Monitor: ASUS ProArt
